README.md

```diff
@@ -224,7 +223,7 @@ https: false
 rules:
   - paths:
     - pathPrefix: /pipeline
-      backend: pipeline-demo' | egctl object create
+      backend: pipeline-demo' | egctl create object 
```

Comment:
Actually, the spec of object is self-explained. And the term `object` is very abstract and confusing, which we discussed long before. So this enhancement would be a very good chance to weaken the concept for users. So I think we could support command interface in these ways:

```bash
$ egctl create -f server.yaml
$ echo 'kind: HTTPServer ...' | egctl create

$ egctl apply -f server.yaml
$ echo 'kind: Pipeline ...' | egctl apply

$ egctl delete httpserver server-proxy
$ egctl delete pipeline pipeline-proxy

$ egctl get httpserver

# show HTTPServer dedicated table ...

$ egctl get pipeline

# show Pipeline dedicated table ...

$ egctl get thirdpartyobject

# show basic info like k8s crd
```

We can also support aliases for frequently-used objects, such as `hs`, `httpservers` for `httpserver`, `pl`, `pipelines` for `pipeline`, etc.

I know the underhood APIs would be the same for all objects, but this design could be a great step to make users feel clear, just like kubectl hides the underhood concept object.